File: LeetCode_3005/Main.cpp
#include<bits/stdc++.h>
using namespace std;

class Solution {
public:
    int maxFrequencyElements(vector<int>& nums) {
        vector<int> cnt(101, 0);
        for (int num : nums) {
            cnt[num] += 1;
        }
        int mx = *max_element(begin(cnt), end(cnt));
        int sum = 0;
        for (int c : cnt) {
            sum = (c == mx ? sum + c : sum);
        }
        return sum;
    }
};

File: LeetCode_3005/Main.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var maxFrequencyElements = function(nums) {
    const cnt = Array(101).fill(0); 

    for (let num of nums) {
        cnt[num]++;
    }

    let mx = Math.max(...cnt);

    let result = 0;
    for (let f of cnt) {
        if (f == mx) {
            result += f;
        }
    }

    return result;
};

console.log(maxFrequencyElements([1,2,2,3,1,4]))
